libs/features/mas/commerce/src/settings.js

```diff
@@ -135,15 +133,18 @@ function getSettings(config = {}) {
     // TODO: add alias names for meta, search and storage
     // See https://git.corp.adobe.com/wcms/tacocat.js/pull/348#discussion_r6557570
     const { commerce = {}, locale = undefined } = config;
-    const hostEnv =
-        config.env?.name === HostEnv.PROD
-            ? HostEnv.PROD
-            : toEnumeration(
-                  getParameter(PARAM_ENV, commerce, { metadata: false }),
-                  HostEnv,
-                  HostEnv.PROD,
-              );
-    const env = hostEnv === HostEnv.STAGE ? Env.STAGE : Env.PRODUCTION;
+    const lowHostEnv = ['local', 'stage'].includes(config.env?.name);
+    let env = Env.PRODUCTION;
+    let wcsURL = 'https://www.adobe.com/web_commerce_artifact';
+    if (lowHostEnv) {
+      wcsURL = 'https://www.stage.adobe.com/web_commerce_artifact';
```

Comment:
Nit: maybe string repetition could be avoided by determining the environment and then using a template literal to form the URL? Or maybe even a simple `replace`?

Reply:
refactored a bit